### DISCUSSION TITLE
where is  the 3.6.5.ez？

### DISCUSSION THREAD

**Discussion by maoxiaoxiong** [rabbitmq/rabbitmq-delayed-message-exchange#166] [General]
I want to get 3.6.5.ez ，but  source i can find 
rabbitmq-delayed-message-exchange-3.6.5.ez

**Comment by michaelklishin** [comment]
We used to have an archive on Bintray but Bintray no longer exists. You can build `3.6.5` using the tag in this repo, RabbitMQ 3.6.5 (which has been [long out of support](https://www.rabbitmq.com/versions.html)) and an Erlang version they [supported](https://www.rabbitmq.com/which-erlang.html).
